pkg/security/probe/resolvers.go

```diff
@@ -89,13 +89,13 @@ func (r *Resolvers) resolveBasename(e *model.FileFields) string {
 }
 
 // resolveFileFieldsPath resolves the inode to a full path
-func (r *Resolvers) resolveFileFieldsPath(e *model.FileFields) (string, error) {
+func (r *Resolvers) resolveFileFieldsPath(e *model.FileFields, ctx *model.PIDContext) (string, error) {
```

Comment:
It seems that only the PID is used and not the whole PIDContext. I would suggest to change the signature to simply pass the PID